static/app/components/globalDrawer/components.tsx

```diff
@@ -70,7 +70,7 @@ interface DrawerHeaderProps {
   hideCloseButton?: boolean;
 }
 
-export const DrawerHeader = forwardRef(function _DrawerHeader(
```

Comment:
the lint rules didn't treat this like a react component, it has to start with a capitalized char